grading/execute.cpp

```diff
@@ -1059,7 +1060,8 @@ void cin_reader(std::mutex* lock, std::queue<std::string>* input_queue, bool* CH
 
 
 // Executes command (from shell) and returns error code (0 = success)
-int execute(const std::string &cmd, 
+int execute(const std::string &cmd,
+      const bool isInstructor,
```

Comment:
I don't like the addition of this parameter to execute.
What is the purpose? 
It is a big change and it should not be necessary.